src/main/java/lotto/domain/Lotto.java

```diff
@@ -26,11 +26,15 @@ public Lotto() {
   }
 
   private void checkDuplicateNumbers(final List<Integer> numberPull) {
-    if(numberPull.size() != numberPull.stream().distinct().count()){
+    if(numberPull.size() != getDistinctNumberPullSize(numberPull)){
       throw new IllegalArgumentException(Message.MSG_ERROR_SAME_LOTTO_NUMBER);
     }
   }
 
+  private long getDistinctNumberPullSize(final List<Integer> numberPull) {
+    return numberPull.stream().distinct().count();
```

Comment:
```suggestion
    return numberPull.stream()
            .distinct()
            .count();
```
이런 식으로 코드를 변경해보면 좋을 것 같아요. :)

메소드를 통해 기능을 분리하는 것도 좋지만
스트림이나 메소드 체이닝을 사용하는 경우에는 줄바꿈을 통해 표현하는 것도 가능합니다.

이렇게 하는 이유는 한 줄에 모든 일을 나열하면 코드가 길어지고,
보기가 어려운 동시에 이해하기가 어려워지기 때문입니다~

Reply:
코드의 가독성에 조금 더 신경쓰란 말씀이시군요.
확인 후 수정하도록 하겠습니다.